### PR TITLE
Add zap clause to LTspice cask

### PR DESCRIPTION
**Pull request by nicholasngai**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name ~~and version~~.
  - Cask has no attached version

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/ltspice.rb [hunk 1]
```diff
@@ -7,4 +7,10 @@
   homepage 'http://www.analog.com/en/design-center/design-tools-and-calculators/ltspice-simulator.html'
 
   app 'LTspice.app'
+
+  zap trash: [
+               '~/Documents/LTspice/examples',
+               '~/Library/Applicaiton Support/LTspice',
```
Comment (adidalal) [inline Casks/ltspice.rb]: typo?

File: Casks/ltspice.rb [hunk 2]
```diff
@@ -7,4 +7,10 @@
...
+             ],
+      rmdir: '~/Documents/LTspice'
```
Comment (adidalal) [inline Casks/ltspice.rb]: Why twice? (once only the subdirectory)?

**Review by adidalal — DISMISSED** [review]

**Comment by nicholasngai** [comment]
Sorry, I wasn’t entirely sure what to do with the examples. LTspices stores user-created schematics in the `~/Documents/LTspice` folder and the pre-packaged examples in the `~/Documents/LTspice/examples` subfolder, so I wasn’t sure if I should trash the examples in the zap stanza. The idea was to trash the examples, and if there were no user-created schematics, it would remove the `~/Documents/LTspice` folder since it would be empty after trashing the examples.

**Comment by adidalal** [comment]
Ok, understood. That's fine, then!

Looks like no in-app updates, though. `http://ltspice.linear-tech.com/LTspiceAppUpdater` and `http://ltspice.analog.com/LTspiceAppUpdater` return an error

**Review by adidalal — APPROVED** [review]

**Pull request merged**